packages/angular/src/schematics/application/application.ts

```diff
@@ -337,6 +337,7 @@ export default function(schema: Schema): Rule {
       : `${options.name}/e2e`;
 
     return chain([
+      ngAdd(options),
```

Comment:
This is taken care of by the following statement

Reply:
Rebased => fixed